frontend/simple/views/Join.vue

```diff
@@ -100,33 +100,30 @@
 import * as Events from '../../../shared/events'
 import backend from '../js/backend/'
 import * as db from '../js/database'
-import {latestContractState} from '../js/state'
 import L from '../js/translations'
 export default {
   name: 'Join',
-  mounted () {
-    this.fetchData()
+  async created () {
+    if (!this.$store.state[this.$route.query.groupId]) {
+      await backend.subscribeAndSync(this.$route.query.groupId, true)
+      for (let member of this.$store.state[this.$route.query.groupId].members) {
+        await backend.subscribeAndSyncIdentity(member, true)
+      }
+    }
```

Comment:
This seems quite out of place. Something as core as subscribing to our members shouldn't be done here, I don't think.